### ISSUE TITLE
Sidebar and dropdown views only show groups 1 level deep

### ISSUE THREAD

**Issue by manastalukdar** [fabiospampinato/vscode-projects-plus#64]
I have nested groups in my config file 4 levels deep. But in the sidebar, the displayed nested in only down to 1 level, and that too if there is at least one project under the root group. The dropdown for selecting project exhibits similar behavior. How do I enable these two pieces to display the nested groups and their projects all the way down?

**Comment by fabiospampinato** [comment]
I can't reproduce, can you share a minimal configuration file that reproduces the issue?

**Comment by manastalukdar** [comment]
@fabiospampinato , here is a config that reproduces the issue:

```json
{
  "groups": [
    {
      "name": "Group1",
      "projects": [
        {
          "name": "wiki",
          "description": "",
          "path": "d:\\test\\wiki"
        },
        {
          "name": "email",
          "path": "d:\\test\\email",
          "description": ""
        }
      ]
    },
    {
      "name": "Group2",
      "groups": [
        {
          "name": "Group2-Subgroup1",
          "groups": [
            {
              "name": "Group2-Subgroup1-Subgroup1",
              "projects": [
                {
                  "name": "some-project1",
                  "description": "dummy project1",
                  "path": "g:\\test\\dummy1"
                }
              ]
            },
            {
              "name": "Group2-Subgroup1-Subgroup2",
              "projects": [
                {
                  "name": "some-project2",
                  "description": "dummy project2",
                  "path": "g:\\test\\dummy2"
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}
```

**Comment by fabiospampinato** [comment]
Fixed in v1.23.3 👍
